src/Diagnostics.RuntimeHost/Middleware/DiagnosticsRequestMiddleware.cs

```diff
@@ -29,8 +28,8 @@ public async Task Invoke(HttpContext httpContext)
         {
             Exception exception = null;
             int statusCode = 0;
-            GenerateMissingRequestHeaders(ref httpContext);
-            BeginRequest_Handle(httpContext);
+            GenerateMissingRequestHeaders(httpContext);
```

Comment:
I think we should be good here, the httpContext itself I think is already being passed by reference to the method, so `GenerateMissingRequestHeaders` will have no problem modifying the http context right?

Reply:
Right, the `ref` was unnecessary since the reference itself is not being reassigned, just the underlying properties.